modules/exploits/linux/redis/redis_unauth_exec.rb

```diff
@@ -0,0 +1,271 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GoodRanking
+
+  include Msf::Exploit::Remote::TcpServer
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::FileDropper
+  include Msf::Auxiliary::Redis
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Redis Unauthenticated Code Execution',
+      'Description'    => %q{
+        This module can be used to leverage the extension functionality added by Redis 4.x and 5.x
+        to execute arbitrary code, to transmit the evil extension it makes use of the feature of Redis
```

Comment:
> evil extension

It's not really evil. :)

Also, `code, to` should be `code. To`.

Reply:
My poor english :)